src/_examples/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Examples
+---
+
+# Module-specific Examples
+
+<ul>
+{% for example in site.examples %}
+  {% if example.module %}
+    <li><a href="{{ example.url }}">{{ example.module }}</a></li>
+  {% endif %}
+{% endfor %}
+</ul>
+
+# Standalone Projects
+
+These projects are using Workbox for their service worker:
+
+- iFixit PWA (`gulp`-based build process)
+[[source](https://github.com/GoogleChrome/application-shell/tree/ifixit-pwa/ifixit-pwa)]
+[[demo](https://ifixit-pwa.appspot.com/)]
+- React HN (`npm scripts`-based build process)
```

Comment:
Can we add images or something as well to these?

I'm imaging the list to look very dull.